specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/networkData.json

```diff
@@ -0,0 +1,361 @@
+            "calculatedDateTime": {
+              "type": "string",
+              "format": "date-time",
+              "readOnly": true,
+              "description": "The UTC time on which the allowed connections was calculated"
+            },
+            "inboundConnectedResources": {
+              "$ref": "#/definitions/ConnectedResources"
+            },
+            "outboundConnectedResources": {
+              "$ref": "#/definitions/ConnectedResources"
+            }
+          }
+        },
+        "actualTraffic": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The actual traffic details",
+          "properties": {
+            "startTime": {
+              "type": "string",
+              "format": "date-time",
+              "readOnly": true,
+              "description": "The UTC start time of the actual traffic time range"
+            },
+            "endTime": {
+              "type": "string",
+              "format": "date-time",
+              "readOnly": true,
+              "description": "The UTC end time of the actual traffic time range"
+            },
+            "trafficDataState": {
+              "type": "string",
+              "description": "Indicate whether the Azure monitor agent is installed on the vm",
+              "enum": [
+                "Available",
+                "Not available"
+              ],
+              "x-ms-enum": {
+                "name": "trafficDataState",
+                "modelAsString": true,
+                "values": [
+                  {
+                    "value": "Available",
+                    "description": "The Azure monitor agent is installed on the vm"
+                  },
+                  {
+                    "value": "Not available",
+                    "description": "the Azure monitor agent is not installed on the vm"
+                  }
+                ]
+              }
+            },
+            "inboundConnectedResources": {
+              "$ref": "#/definitions/ActualTrafficConnectedResources"
+            },
+            "outboundConnectedResources": {
+              "$ref": "#/definitions/ActualTrafficConnectedResources"
+            }
+          }
+        },
+        "unusedPorts": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The unused ports details",
+          "properties": {
+            "inboundConnectedResources": {
+              "$ref": "#/definitions/ConnectedResources"
+            },
+            "outboundConnectedResources": {
+              "$ref": "#/definitions/ConnectedResources"
+            }
+          }
+        }
+      }
+    },
+    "ConnectedResources": {
+      "readOnly": true,
+      "type": "object",
+      "description": "The connections details",
+      "properties": {
+        "tcpPortsCount": {
+          "readOnly": true,
+          "type": "integer",
+          "description": "The allowed connections tcp ports count"
```

Comment:
>allowed connections [](start = 30, length = 19)

this definition is used in "unusedPorts" as well

Reply:
fixed.